broker-cli/proto/broker.proto

```diff
@@ -883,6 +879,14 @@ message Capacity {
   string inactive_send_capacity = 6;
   // Total inactive amount a user can send in the currency. This is currently in base units (e.g. Satoshis/Litoshis)
   string inactive_receive_capacity = 7;
+  // Total available amount a user can send in the currency. This is currently in base units (e.g. Satoshis/Litoshis)
+  string available_send_capacity = 8;
+  // Total available amount a user can send in the currency. This is currently in base units (e.g. Satoshis/Litoshis)
+  string available_receive_capacity = 9;
+  // Total outstanding amount a user can send in the currency. This is currently in base units (e.g. Satoshis/Litoshis)
```

Comment:
this documentation needs to change to reflect outstanding balance. We could simply write `Total outbound outstanding amount`? or something. Documentation below needs to be updated too